packages/salesforcedx-apex-debugger/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "outDir": "out",
     "preserveConstEnums": true,
     "strict": true,
-    "strictPropertyInitialization": false
+    "strictPropertyInitialization": true
```

Comment:
I think we might be able to just remove this setting. The default inherited setting is to be true.

Reply:
CC: @allileong

Reply:
ok.  I'll make that change.

Reply:
done